gdcmodels/__init__.py

```diff
@@ -11,6 +11,20 @@ def load_yaml(filename):
         return yaml.safe_load(f)
 
 
+def process_ref(name, reference, path):
+    ref = reference['$ref']
```

Comment:
this might break for `*_centric` mappings

Reply:
humm not sure I see how. right now it is running through them without a blowing up... maybe we can sync up offline

Reply:
oh, you are right, there's that `isfile` check. my bad.